src/info/guardianproject/otr/app/im/app/AccountWizardActivity.java

```diff
@@ -238,28 +244,42 @@ public boolean onPrepareOptionsMenu(Menu menu) {
 
     private void buildAccountList ()
     {
+        int i = 0;
+        int accountProviders = 0;
         List<String> listProviders = helper.getProviderNames();
 
         mGoogleAccounts = AccountManager.get(this).getAccountsByType(GTalkOAuth2.TYPE_GOOGLE_ACCT);
 
-        mAccountList = new String[listProviders.size()+3][2]; //potentialProviders + google + create account + burner
-
-        int i = 0;
-
-        mAccountList[i][0] = getString(R.string.i_want_to_chat_using_my_google_account);
-        mAccountList[i++][1] = getString(R.string.account_google_full);
+        if (mGoogleAccounts.length > 0) {
```

Comment:
does this mean you can't connect to gtalk via chatsecure if you don't have a google account setup at the system level ?

If using cyanogenmod without any google account for example, it would be convenient to still be able to connect to gtalk via chatsecure.